uttut/tests/test_expand_by_entities.py

```diff
@@ -41,34 +48,46 @@ def setUp(self):
         ]
         self.expected_entity_lists = [
             [
-                Entity(name='日期', value='明天', start=3, end=5, replacements=['下禮拜二']),
+                Entity(name='日期', value='明天', start=3,
+                       end=5, replacements=['下禮拜二']),
                 Entity(name='出發地', value='紐約', start=6, end=8),
-                Entity(name='目的地', value='新加坡', start=10, end=13, replacements=['斯堪地那維亞', 'KIX']),
+                Entity(name='目的地', value='新加坡', start=10,
+                       end=13, replacements=['斯堪地那維亞', 'KIX']),
             ],
             [
-                Entity(name='日期', value='下禮拜二', start=3, end=7, replacements=['明天']),
+                Entity(name='日期', value='下禮拜二', start=3,
+                       end=7, replacements=['明天']),
```

Comment:
Avoid coding style that is inconsistent with our repo

Reply:
e1ac824